docs/howto/custom-rootfs-multiple-targets.md

```diff
@@ -0,0 +1,314 @@
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+LOGIN_RESPONSE=$( curl -s -X POST "$LANDSCAPE_URL/api/v2/login" \
+    --data "$LOGIN_JSON"                                        \
+    --header "Content-Type: application/json"                   \
+    --header "Accept: application/json" )
+
+JWT=$( echo $LOGIN_RESPONSE | jq .token | tr -d '"')
+```
+````
+
+````{group-tab} PowerShell
+```powershell
+$LOGIN_RESPONSE = Invoke-WebRequest -Method POST `
+    -URI "$LANDSCAPE_URL/api/v2/login"           `
+    -Body "$LOGIN_JSON" -ContentType "application/json"
+
+$JWT = ConvertTo-SecureString -AsPlainText -Force $( $LOGIN_RESPONSE.Content | ConvertFrom-Json).token
+```
+````
+
+`````
+
+## Send the Install request
+
+Build the payload with information about the WSL instance to be deployed. In this case it would look like
+`{"rootfs_url": "http://landscape.mib.com:9009/ubuntu-24.04-custom.tar.gz", "computer_name": "Carbonizer", "cloud_init": <base64 encoded material>}`.
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+WSL_JSON=$( jq -n                           \
+    --arg rf "$ROOTFS_URL"                  \
+    --arg cn "$COMPUTER_NAME"               \
+    --arg b64 "$BASE64_ENCODED_CLOUD_INIT"  \
+    '{rootfs_url: $rf, computer_name: $cn, cloud_init: $b64}' )
+```
+````
+
+````{group-tab} PowerShell
+```powershell
+$WSL_JSON = @{
+ rootfs_url = "$ROOTFS_URL"
+ computer_name = "$COMPUTER_NAME"
+ cloud_init = "$BASE64_ENCODED_CLOUD_INIT"
+} | ConvertTo-Json
+
+```
+````
+`````
+
+At the moment of this writing there is no specific API endpoint to trigger
+installation of WSL instances on multiple Windows machines at once.
+Instead we send one request per target machine.
+
+`````{tabs}
+
+````{group-tab} Bash
+```bash
+for COMPUTER_ID in "${PARENT_COMPUTER_IDS[@]}"; do
+    API_RESPONSE=$( curl -s -X POST                             \
+        "$LANDSCAPE_URL/api/v2/computers/$COMPUTER_ID/children" \
+        --data "$WSL_JSON"                                      \
+        --header "Authorization:Bearer $JWT"                    \
+        --header "Content-Type: application/json"               \
+        --header "Accept: application/json" )
+
+    # show the response
+    echo $API_RESPONSE
+    echo
+done
+```
+````
+
+````{group-tab} PowerShell
+
+```powershell
+foreach ($COMPUTER_ID in $PARENT_COMPUTER_IDS) {
+    $API_RESPONSE = Invoke-WebRequest -Method POST -Body "$WSL_JSON" `
+        -Uri "$LANDSCAPE_URL/api/v2/computers/$COMPUTER_ID/children" `
+        -Authentication Bearer -Token $JWT -ContentType "application/json"
+
+    # show the response
+    Write-Output $API_RESPONSE
+}
+```
+````
+`````
+
+When that completes, you'll be able to find activities in the Landscape
+dashboard about the installation of a new WSL instance for each of the Windows
+machines listed.
+
+## Summarising the steps in a single script
+
+The steps above can be made into a single script:
```

Comment:
For the script to run successfully the environment variables would still need to be defined.
It might be worth noting before/after showing the complete script that the vars need to be defined in case anyone skips to the part of the guide with the complete script.